vendor/assets/javascripts/util.js

```diff
@@ -0,0 +1,7 @@
+Cmsify.util = {
+  filterAuthenticityToken: function(arr) {
+    return arr.filter(function(element) {
+      return element.name !== "authenticity_token";
+    });
+  }
+}
```

Comment:
I'm getting rid of it because it changes between the time that the js first runs and the time the form gets submitted, I think it might be a time based thing, I don't remember how the Authenticity token works in rails. Anyway, it was basically causing the form to always register as having been modified because the value of the token changes. Since we don't actually need the token for this use case, just the user changed fields I got rid of it rather than trying to account for it in a different way.